src/app/public/modules/lookup/lookup.component.ts

```diff
@@ -229,12 +229,24 @@ export class SkyLookupComponent
   @ViewChild(SkyAutocompleteComponent)
   private autocompleteComponent: SkyAutocompleteComponent;
 
+  @ViewChild('buttonsTemplateRef', {
+    read: TemplateRef,
+    static: true
+  })
+  private buttonsTemplateRef: TemplateRef<any>;
```

Comment:
If we do decide to change the template tag names we should probably also change these fields' names.

Reply:
Updated.